src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/InterceptorConfig.java

```diff
@@ -25,12 +25,14 @@ public class InterceptorConfig implements WebMvcConfigurer {
     static final String DETAILS = "/private/**/details";
     static final String MANAGING_OFFICERS_PRIVATE_DATA = "/private/**/managing-officers";
     static final String BENEFICIAL_OWNERS_PRIVATE_DATA = "/private/**/beneficial-owners";
+    static final String TRUST_DETAILS_PRIVATE_DATA = "/private/**/trusts/details";
```

Comment:
covered by line 25 but I'm thinking we could have one rule for all the trusts endpoints?
static final String TRUST_PRIVATE_DATA = "/private/**/trusts/**";
otherwise we are going to end up with 4 items.

Reply:
I'm assuming you mean `TRUST_PRIVATE_DATA = "/private/*/trusts/*;` I'll do that now

Reply:
Good point, so i guess it will look like /private/**/trusts/** ?